crates/shared/src/order_quoting.rs

```diff
@@ -195,6 +197,46 @@ impl TryFrom<QuoteRow> for QuoteData {
             // Even if the quote was verified at the time of creation
             // it might no longer be accurate.
             verified: false,
+            interactions: vec![],
+        })
+    }
+}
+
+impl TryFrom<database::quotes::QuoteWithInteractions> for QuoteData {
+    type Error = anyhow::Error;
+
+    fn try_from(input: database::quotes::QuoteWithInteractions) -> Result<QuoteData> {
```

Comment:
Either this:
```suggestion
    fn try_from((quote, interaction): database::quotes::QuoteWithInteractions) -> Result<QuoteData> {
```
or use a regular QuoteWithInteractions struct with named arguments.

Reply:
`QuoteWithInteractions` has been refactored to contain same fields as `Quote` struct + interactions vector.